app/react/settings/users/userContainer.jsx

```diff
@@ -0,0 +1,76 @@
+import React from "react";
+import PropTypes from "prop-types";
+// import { connect } from "react-redux";
+import { bindActionCreators } from "redux";
+import connectWithStore from "react/react-transition/connectWithStore";
+import * as UsersActions from "react/common/state/users/users.actions";
+import * as RolesActions from "react/common/state/roles/roles.actions";
+import {
+    isLoadingRoles,
+    getRoles
+    // getRoleError
+} from "react/common/state/roles/roles.reducers";
+import { getUsers } from "react/common/state/users/users.reducers";
+import UserForm from "react/settings/users/UserForm";
+
+class UserContainer extends React.Component {
+    componentDidMount() {
+        // Need UI to handle failures for this.
+        // Error currently saved to state.error
+        this.props.RolesActions.requestRoles();
+    }
+
+    render() {
+        return (
+            <main role="main">
+                <div>
+                    <h2>Add people to Ushahidi</h2>
+                    <p>Add members of your team, etc.. etc...</p>
```

Comment:
Is there real text for this yet?

Reply:
nope. designs were only finished today I think. need to update where I can and we can discuss the more complex pieces of UI at the retreat and will create an issue